tests/expressions/tests.py

```diff
@@ -2333,10 +2334,46 @@ def test_resolve_output_field(self):
                 expr = Value(value)
                 self.assertIsInstance(expr.output_field, output_field_type)
 
-    def test_resolve_output_field_failure(self):
-        msg = "Cannot resolve expression type, unknown output_field"
-        with self.assertRaisesMessage(FieldError, msg):
-            Value(object()).output_field
+    def test_output_field_resolution(self):
+        # why does this fail, but test_resolve_output_field_with_null pass?
```

Comment:
For my own understanding - to understand the difference between this test and the set of tests containing test_resolve_output_field_with_null

Reply:
Still reading https://code.djangoproject.com/ticket/33397 and https://github.com/django/django/pull/15271 Q__Q